Data/Alias/alias.txt

```diff
@@ -126,10 +123,10 @@
 .ils07C From LIMES via TONIC, cleared ILS RWY 07C approach.
 .ils07r From LIMES via STELA, cleared ILS RWY 07R approach.
 .ils07l From LIMES via TUTBA, cleared ILS RWY 07L approach.
-.ils25l From TD, cleared ILS RWY 25L approach. 
-.ils25C From TD, cleared ILS RWY 25C approach. 
-.tra25R From TD, descend via RNAV transition.
-.ils25R From TOPUN, cleared ILS RWY 25R approach.
+.ils25l From RIVMI, cleared ILS approach RWY 25L. 
+.ils25C From RIVMI, cleared ILS approach RWY 25C. 
+.tra25R From RIVMI, descend via RNAV transition.
+.ils25R From TOPUN, cleared ILS approach RWY 25R.
```

Comment:
If the correct phraseology is to place the runway after the approach, then the 07 aliases also need changing